lib/spec/openapi/utils.js

```diff
@@ -323,20 +323,30 @@ function resolveResponse (fastifyResponseJson, produces, ref) {
 
     // add schema when type is not 'null'
     if (rawJsonSchema.type !== 'null') {
-      const content = {}
+      if (Array.isArray(resolved.contentTypes)) {
+        const content = {}
+        for (const contnetTypeResponse of resolved.contentTypes) {
+          const media = schemaToMedia(contnetTypeResponse.schema)
+          content[contnetTypeResponse.content] = media
+          response.content = content
+        }
```

Comment:
do you prefer using reduce here, or it's fine?

Reply:
I ❤️ functional programming (aka reduce) but if you just move line 331 you won't be mutating the `response` object every time which IMO should be fine as you would just be building the `content` obj.

Also you misspelled `contnetTypeResponse` 📝

Reply:
just realized this is outdated 🤦

Reply:
so this can be resolved, right?